routers/users.py

```
@@ -0,0 +1,44 @@
+            status_code=status.HTTP_409_CONFLICT,
+            detail="이미 사용 중인 이메일입니다."
+        )
+
+    # 3. 새로운 유저 객체 생성
+    new_user = {
+        "id": generate_id("user", users),
+        "email": user.email,
+        "nickname": user.nickname,
+        "profile_image_url": user.profile_image_url,
+        "password": user.password,  # 2단계에서 해싱 처리 예정
+        "created_at": datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)